tests/ModelContextProtocol.AspNetCore.Tests/StatelessServerTests.cs

```diff
@@ -184,7 +184,7 @@ public static async Task<string> TestSamplingErrors(IMcpServer server)
         var asSamplingChatClientEx = Assert.Throws<InvalidOperationException>(() => server.AsSamplingChatClient());
         Assert.Equal(expectedSamplingErrorMessage, asSamplingChatClientEx.Message);
 
-        var requestSamplingEx = await Assert.ThrowsAsync<InvalidOperationException>(() => server.RequestSamplingAsync([]));
+        var requestSamplingEx = await Assert.ThrowsAsync<InvalidOperationException>(() => server.SampleAsync([]));
```

Comment:
Can we add a test similar to this one for `ElicitAsync`? I think it should pass since you added ThrowIfElicitationUnsupported.